worker/uniter/pebblenotices.go

```diff
@@ -172,12 +184,16 @@ func (n *pebbleNoticer) processNotice(containerName string, notice *client.Notic
 		// of hooks as this reflects a change of (workload) state, rather than
 		// a change of data. See OP046 for more background.
 		switch {
-		case kind == "perform-check" && chg.Status == "Error":
+		case kind == "perform-check" && (chg == nil || chg.Status == "Error"):
 			eventType = container.CheckFailedEvent
-		case kind == "recover-check" && chg.Status == "Done":
+		case kind == "recover-check" && (chg == nil || chg.Status == "Done"):
 			eventType = container.CheckRecoveredEvent
 		default:
-			n.logger.Debugf("container %q: ignoring %s, status %s", containerName, kind, chg.Status)
+			chgStatus := "<unknown>"
```

Comment:
Should this be `<not found>`

Reply:
Not really. The *change* was not found, but we're showing *status* here, so I think "status <unknown>" fits better.